config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   devise_for :users
+  get "/pages/:page" => "pages#show"
```

Comment:
em dùng cú pháp mới hơn, viết như này xem có được ko nhé:
`get "/pages/:page"`, to: "pages#show"

Reply:
vâng a.